src/atsamd/chipid.c

```diff
@@ -6,12 +6,16 @@
 
 #include "autoconf.h" // CONFIG_USB_SERIAL_NUMBER_CHIPID
 #include "generic/canserial.h" // canserial_set_uuid
-#include "generic/usb_cdc.h" // usb_fill_serial
-#include "generic/usbstd.h" // usb_string_descriptor
 #include "sched.h" // DECL_INIT
 
 #define CHIP_UID_LEN 16
 
+#if CONFIG_HAVE_SAMD_USB
```

Comment:
Is this ifdef necessary?  CONFIG_USB_SERIAL_NUMBER_CHIPID should only be set if CONFIG_USB is set, so this should all get compiled out without these ifdefs..

Reply:
Hmm, yes - it should get compiled out. I probably just added guards knowing that USB is not available on this chip. I am hesitant to remove it though, for code clarity reasons.

Reply:
Opted to remove.